src/main/java/Task.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Represents a Task. Contains a Task constructor, two methods to mark and unmark tasks, toString() method as well as a isMark() method to check if Task is marked
+ */
+public class Task {
+    private boolean mark;
```

Comment:
Not sure I like the naming of the boolean variable. As much as possible, use a prefix such as is, has, was, etc. for boolean variable/method names so that linters can automatically verify that this style rule is being followed. Perhaps it would be better to name it as follows?
```suggestion
    private boolean isMarked;
```